docs/kqueen.rst

```diff
@@ -442,6 +442,42 @@ Provision a Kubernetes cluster using Google Kubernetes Engine
 #. Click ``Submit``.
 #. To track the cluster status, navigate to the KQueen main dashboard.
 
+Provision a Kubernetes cluster using Openstack Kubespray Engine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Log in to Openstack Horizon Dashboard.
+#. Select your Project.
+#. Navigate to Project`-> ``Compute`` -> ``Access & Security`` -> ``API Access`` tab and click
+   ``Download OpenStack RC File``.
+#. Log in to the KQueen web UI.
+#. From the ``Create Provisioner`` page, select ``Openstack Kubespray Engine``.
+#. Specify provisioner requirements with the Openstack RC File, downloaded earlier.
+#. Click ``Deploy Cluster``.
+#. Select the defined Openstack provisioner.
+#. Specify the cluster requirements, and pay attention on the following cases:
+
+   #. ``Ssh key name`` is name of SSH key pair. You should choose from existing pairs or create a new one.
+   #. ``Flavor`` must be at least ``m1.small`` (2GB RAM, dual-core CPU).
+   #. ``Ssh username`` is login username for nodes. It depends on defined ``Image name``. 
```

Comment:
pls capitalize  the remaining letters, I mean ``SSH username``

Reply:
The phrase "It depends on defined Image name" seems incorrect for me and confuses. I think "It depends on the defined image" is more suitable